mypy/suggestions.py

```diff
@@ -928,7 +937,8 @@ def refine_type(ti: Type, si: Type) -> Type:
     s = get_proper_type(si)
 
     if isinstance(t, AnyType):
-        return s
+        # If s is also an any, we return t in case it is a missing_import any
+        return s if not isinstance(s, AnyType) else t
```

Comment:
But what if `t` was a "plain" `Any` and `s` is a missing import? Is such case possible?